bazel/emscripten_toolchain/wasm_cc_binary.bzl

```diff
@@ -32,12 +32,13 @@ def _wasm_transition_impl(settings, attr):
 
     return {
         "//command_line_option:compiler": "emscripten",
-        "//command_line_option:crosstool_top": "@emsdk//emscripten_toolchain:everything",
```

Comment:
`crosstool_top` stopped doing things in Bazel 7 without a flag: https://github.com/bazelbuild/bazel/issues/21121